### ISSUE TITLE
[Node/meteor] Not compatible with meteor 2.0

### ISSUE THREAD

**Issue by cyrille-sg** [Scalingo/nodejs-buildpack#41]
It seems that with meteor 2.0, the building process fail in the node-pre-gyp:
```
   npm ERR! argv "node" "/build/9329c3e7-a984-4f67-9c80-7869c955ee65/.scalingo/node/bin/npm" "install" "--production=false" "--unsafe-perm" "--userconfig" "/build/9329c3e7-a984-4f67-9c80-7869c955ee65/.npmrc"
       npm ERR! node v0.10.48
       npm ERR! npm  v3.10.10
       npm ERR! code ELIFECYCLE
       
       npm ERR! bcrypt@5.0.0 install: `node-pre-gyp install --fallback-to-build`
       npm ERR! Exit status 8
       npm ERR! 
       npm ERR! Failed at the bcrypt@5.0.0 install script 'node-pre-gyp install --fallback-to-build'.
       npm ERR! Make sure you have the latest version of node.js and npm installed.
       npm ERR! If you do, this is most likely a problem with the bcrypt package,
       npm ERR! not with npm itself.
       npm ERR! Tell the author that this fails on your system:
       npm ERR!     node-pre-gyp install --fallback-to-build
       npm ERR! You can get information on how to open an issue for this project with:
       npm ERR!     npm bugs bcrypt
       npm ERR! Or if that isn't available, you can get their info via:
       npm ERR!     npm owner ls bcrypt
       npm ERR! There is likely additional logging output above.
```

**Comment by timothyarmes** [comment]
Hi. Can someone please give me a timescale on this fix?

**Comment by Soulou** [comment]
Hi @timothyarmes the fix has been scheduled in the sprint of the 2 coming weeks

**Comment by Soulou** [comment]
So from now and the 5th of February

**Comment by EtienneM** [comment]
Fixed in #42
